devicetypes/Orvibo/Orvibo-Motion-Sensor.src/Orvibo-Motion-Sensor.groovy

```diff
@@ -0,0 +1,126 @@
+/**
+ *  Orvibo Motion Sensor
+ *
+ *  Copyright 2016 Chris Kitch
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+    definition (name: "Orvibo Motion Sensor", namespace: "Orvibo", author: "jk0218.zhang@samsung.com", vid: "generic-motion-2", ocfDeviceType: "x.com.st.d.sensor.motion") {
```

Comment:
If the DTH was written in-house, the `namespace` and `author` should be `"smartthings"` and '"SmartThings"`.

Also the file should be moved to the `deviceTypes/smartthings` directory.